scripts/migrate_logs_fix.py

```diff
@@ -0,0 +1,50 @@
+import sys
+import logging
+
+from website.app import init_app
+from website.models import NodeLog
+from scripts import utils as script_utils
+from modularodm import Q
+
+logger = logging.getLogger(__name__)
+
+
+def do_migration(records, dry=False):
+    for log in records:
+        logger.info(
+            'Migrating log - {}, '.format(log._id)
+        )
+        count = 0
+        if not dry:
```

Comment:
A dry run of this script does nothing - if we need the ability to dry run, we'll need to move the `save()` calls to a conditional and keep the rest outside. Otherwise, let's remove the ability to call it with a "dry" param.